app/src/main/java/com/shagalalab/constitution/list/adapter/ItemViewHolder.kt

```diff
@@ -0,0 +1,24 @@
+package com.shagalalab.constitution.list.adapter
+
+import android.view.View
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.shagalalab.constitution.R
+import com.shagalalab.constitution.list.model.Item
+
+class ItemViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    var title: TextView = itemView.findViewById(R.id.title_text)
+    var description: TextView = itemView.findViewById(R.id.description_text)
```

Comment:
You don't need to use `findViewById` in kotlin, remove them, and use the followings in `populateModel`:
- `itemView.title_text`
- `itemView.description_text`